app/controllers/questions_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     @answer = @question.answers.new
   end
 
+  def destroy
+    @question = Question.find(params[:id])
+    if @question.delete
+      flash[:success] = "Your question was deleted successfully!"
+      redirect_to root_path
+    else
+      flash[:danger] = "Failed to delete the question"
```

Comment:
I've never used that method, so I'm uncertain on it's usage, but it sounds appropriate.